hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/file/negative/DeleteFailuresSpec.java

```diff
@@ -51,6 +52,7 @@ private HapiSpec handleRejectsMissingFile() {
                 .then(fileDelete("1.2.3").signedBy(GENESIS).hasKnownStatus(ResponseCodeEnum.INVALID_FILE_ID));
     }
 
+    @HapiTest
```

Comment:
I put `@HapiTest` on the above test(handleRejectsMissingFile) and it's passing. Not sure if we want to add it as well.

Reply:
I'm not sure as well, but based on what @rbair23 mentioned on the call to leave it as is and just whatever is enabled but I'm happy to check with him again today